setup.py

```diff
@@ -44,7 +44,7 @@
     "duckdb-engine",
     "pyodbc",
     # sql.plot module tests
-    "matplotlib",
+    "matplotlib==3.7.2",
```

Comment:
Is there an issue for changing matplotlib version?

Reply:
The latest version of matplotlib breaks the test_magic_plot tests. I discussed this with Eduardo in #878 ([here](https://github.com/ploomber/jupysql/pull/878#issuecomment-1721974298)). I opened an issue regarding that as well (#884).

Reply:
I think it can be added in the changelog with issue number

Reply:
Should I do it here or in #878 ?

Reply:
i think we can leave it out . changelog not required